docs/users-manual/submitting-a-job.rst

```diff
@@ -1182,6 +1190,13 @@ as configured by the administrator of the Vault server:
 
     use_oauth_services = dune_production
 
+.. warning::
+
+   Note that if a handle is not used, the permissions granted by the token will
+   be the default permissions, which is usually the maximal, most permissive
+   set.  Using a handle allows the user to reduce the scope of the permissions
+   granted by the token.
```

Comment:
Yeah, I'm a fan of at least committing what we do know into the docs, even if it isn't perfect.  Hopefully this will a) help someone else, and b) help us not to forget wisdom we've earned at great cost.